assets/js/script.js

```diff
@@ -1,11 +1,76 @@
-document.querySelectorAll('a[href^="#"]')
-    .forEach(function(anchor) {
-        anchor.addEventListener('click', function (e) {
-            e.preventDefault();
+function smoothScroll() {
+    document.querySelectorAll('a[href^="#"]')
+        .forEach(function (anchor) {
+            anchor.addEventListener('click', function (e) {
+                e.preventDefault();
 
-            document.querySelector(this.getAttribute('href')).scrollIntoView({
-                behavior: 'smooth'
+                document.querySelector(this.getAttribute('href')).scrollIntoView({
+                    behavior: 'smooth'
+                });
             });
         });
+} smoothScroll();
+
+function MouseRollover(MyImage) {
```

Comment:
Nazwy parametrów piszemy z małej - wyjątkiem jest sytuacja, w której pod tym parametrem kryłaby się funkcja konstruktora (w Reakcie, w składni JSX będzie to miało znaczenie)